ironfish-cli/scripts/import-export-test.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
```

Comment:
If you need something installed like `expect`, our other commands make that obvious with a nice error by checking if the command is installed.

https://github.com/iron-fish/ironfish/blob/master/ironfish-cli/scripts/build.sh#L11